bundle.yaml

```diff
@@ -437,6 +436,21 @@ spec:
         secret:
           defaultMode: 420
           secretName: webhook-server-cert
+      topologySpreadConstraints:
+      - labelSelector:
+          matchLabels:
+            app.kubernetes.io/component: kubemod-operator
+            control-plane: controller-manager
+        maxSkew: 1
+        topologyKey: topology.kubernetes.io/zone
+        whenUnsatisfiable: ScheduleAnyway
+      - labelSelector:
+          matchLabels:
+            app.kubernetes.io/component: kubemod-operator
+            control-plane: controller-manager
+        maxSkew: 1
+        topologyKey: kubernetes.io/hostname
+        whenUnsatisfiable: DoNotSchedule
```

Comment:
You do not have redundancy if kubemod is on the same node anyway. Therefore I would rather have some pod in pending state which also indicate a problem instead of have a false security feeling with two pods running on the same node.

Reply:
You do have a point regarding redundancy not being there when running multiple pods on the same node.
Another reason to run multiple KubeMod pods on the same node in the past was to reduce latency on the webhook, but this is now resolved with CPU limits removed.
My only remaining concern is that this is a breaking change for those that have deployed KubeMod in multi-replica scenarios. I will document that as a breaking change.
Thanks for your contribution!